docs/_static/custom.css

```diff
@@ -1,14 +1,30 @@
-div.body table.widget-descriptions td p
-{
-    text-align: center;
+body {
+    --color-beta: #c8c8c8;
+    --color-beta-dark: #484848;
+}
+body[data-theme="dark"] {
+    --color-beta: var(--color-beta-dark);
 }
-div.body table.widget-support th p,
-div.body table.widget-support td p {
+@media (prefers-color-scheme: dark) {
+    body:not([data-theme="light"]) {
+        --color-beta: var(--color-beta-dark);
+    }
+}
+
+div.widget-descriptions td p,
+div.widget-support td p {
     text-align: center;
 }
 
-div.body table.widget-descriptions td:first-of-type p,
-div.body table.widget-descriptions th:first-of-type p
-{
+div.widget-descriptions th:first-of-type p {
     text-align: left;
 }
```

Comment:
These rules weren't previously working because the CSS class was on the `div`, not the `table`.

Reply:
Actually, the reason they weren't working was that there was no `div.body`. The CSS class is on both elements, but in case this changes in the future, I've now removed the element name, leaving only the class.